lib/py/setup.py

```diff
@@ -23,7 +23,7 @@
         "baseplate>=1.5,<3.0",
         "pyjwt>=2.0.0,<3.0",
         "thrift-unofficial>=0.14,<1.0",
-        "cryptography>=3.0,<4.0",
```

Comment:
there might be a reason we had `<4.0` before (e.g. breaking changes, etc.)?

but if you verified that the newest version does not really break anything we care about, this is fine.

Reply:
Any advise on what to test?

The main reason for the `<4.0` as far as I can tell is that it was expected to be the next break version but since they change the version scheme that never happened. In fact the version jump from 3.4 to 35

Reply:
honestly I don't know. I don't have any baseplate.py services to test it.

I would imagine have a baseplate.py service to upgrade `cryptography` to `35.x` and then verify (on snoodev maybe) things are not broken?